workspaces/global-header/plugins/global-header/src/hooks/useApplicationLauncherDropdownMountPoints.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defaultApplicationLauncherDropdownMountPoints } from '../defaultMountPoints/defaultMountPoints';
+import { ApplicationLauncherDropdownMountPoint, ScalprumState } from '../types';
+import { useScalprum } from '@scalprum/react-core';
+
+export const useApplicationLauncherDropdownMountPoints = ():
+  | ApplicationLauncherDropdownMountPoint[]
+  | undefined => {
+  const scalprum = useScalprum<ScalprumState>();
+
+  const applicationLauncherDropdownMountPoints =
+    scalprum?.api?.dynamicRootConfig?.mountPoints?.[
+      'global.header/application-launcher'
+    ];
+
+  // default application launcher dropdown components for dev env
+  if (Object.keys(scalprum?.api || {}).length === 0) {
+    return defaultApplicationLauncherDropdownMountPoints;
+  }
```

Comment:
100% fine for now because it's aligned with the existing code.

But I would prefer if we move the "dev env" code in the future into the dev folder.

Reply:
Agreed, we can move this into dev folder later.